app/models/scholarship.rb

```diff
@@ -1,3 +1,5 @@
 class Scholarship < ApplicationRecord
   has_many :scholarship_applications
+
+  scope :unexpired, -> { where('close_time > ? OR close_time IS NULL', Time.now) }
```

Comment:
What do you think about `Time.current` here instead of `Time.now`? The ancient Rails maxims advise against `Time.now` because it is unaware of time zones.

From a business perspective, I'm not sure hour-precision is necessary for scholarship closing times, but I remember my experiences submitting things and they were usually last minute.